app/controllers/api/v2/template_invocations_controller.rb

```diff
@@ -0,0 +1,46 @@
+module Api
+  module V2
+    class TemplateInvocationsController < ::Api::V2::BaseController
+      include ::Api::Version2
+      include ::Foreman::Renderer
+
+      before_action :find_job_invocation, :only => %w{template_invocations}
+
+      api :GET, '/job_invocations/:id/template_invocations',
+          N_('List template invocations belonging to job invocation')
+      param_group :search_and_pagination, ::Api::V2::BaseController
+      param :id, :identifier, :required => true
+      def template_invocations
```

Comment:
why not calling it `index`, which is the standard name for method listing all the resources in rails?

Reply:
Perhaps it was moved from `JobInvocationController`?

Reply:
Is it still an `index` when it is by default scoped by another resource? I would expect `index` to return all template invocations (with possible search), not only template invocations belonging to a certain job invocation